### ISSUE TITLE
NTR: N-carbamyl-beta-aminoisobutyric acid high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8854]
New term request N-carbamyl-beta-aminoisobutyric acid high in urine (elthat_111121051955):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000623
- label: Elevated urinary N-carbamyl-beta-aminoisobutyric acid level
- parents: [Elevated urinary monocarboxylic acid level (HP:0033303)](https://hpo.jax.org/app/browse/term/HP:0033303)
- pmid: 
- def: The amount of N-carbamyl-beta-aminoisobutyric acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8854
- synonyms:
